Dockerfile

```diff
@@ -65,8 +65,8 @@ while [ $attempt -lt $max_attempts ]; do\n\
     exit 1\n\
   fi\n\
   # Use timeout to prevent indefinite hanging if DB is not ready\n\
-  if PGPASSWORD=$DB_PASSWORD psql -w -h $DB_HOST -U $DB_USER -d $DB_NAME -c "ALTER ROLE warranty_user WITH SUPERUSER;" 2>/dev/null; then\n\
```

Comment:
This check seems to be redundant as you need to be SUPERUSER to perform this action in first place, even when the db_user was hardcoded to warranty_user as before you also neded to have your pg user set to warranty_user for the setup to work anyway.